Makefile

```diff
@@ -70,16 +70,16 @@ else
 ifeq ($(LANG_TYPE),RTL)
 	NORM_MODE =3
```

Comment:
@Shreeshrii, where does this norm mode come from? Should it be `NORM_MODE = 2`?

Reply:
```
 unicharset_extractor

Usage: unicharset_extractor [--output_unicharset filename] [--norm_mode mode] box_or_text_file [...]
Where mode means:
 1=combine graphemes (use for Latin and other simple scripts)
 2=split graphemes (use for Indic/Khmer/Myanmar)
 3=pure unicode (use for Arabic/Hebrew/Thai/Tibetan)
Reads box or plain text files to extract the unicharset.

```

Reply:
Strange. That code was added by Ray in commit https://github.com/tesseract-ocr/tesseract/commit/a912967cc. The same commit also added the code which sets a different value in `src/training/language-specific.sh`. There was even a comment from you asking why the value was not set to 3.

Also strange: `unicharset_extractor --help` shows a different help text.

Reply:
One way to check will be to look at the lstm-unicharset files in `ara.traineddata` and other RTL languages.

Reply:
For my test data (Arabic handwriting, about 2500 lines) it did not make a difference whether `NORM_MODE` was 2 or 3. This can be explained by the Tesseract code which increments values below 3, so 2 becomes 3 during the execution.